modules/@apostrophecms/login/ui/apos/components/TheAposLoginHeader.vue

```diff
@@ -89,15 +89,16 @@ export default {
 
   &__header--tiny {
     flex-direction: row;
+    // stylelint-disable-next-line scale-unlimited/declaration-strict-value
     color: #F8F9FA;
 
     .apos-login__project {
-      opacity: 0.7
+      opacity: 0.7;
     }
 
     .apos-login__project-name {
+      // stylelint-disable-next-line scale-unlimited/declaration-strict-value
```

Comment:
ℹ️ I think that we can ignore the warning for now, since they are specific values and I don't want to risk breaking anything